webhooks/README.md

```diff
@@ -72,70 +73,40 @@ curl -X POST \
   -d '{
     "event": "transaction.auth",
-    "url": "https://realtime.com",
+    "url": "https://example.com",
     "headers": {
       "Custom-Header": "my-custom-header"
     }
   }'
 ```
 
-Up to 5 custom headers per webhook can be defined and they need to follow strict character validation patterns. You can find more information regarding validation in the [Webhooks API](https://reference.fidel.uk/v1/reference#create-webhook-brand) reference.
+To delete custom headers from a registered webhook, use the [Update Hooks](https://reference.fidel.uk/reference#update-webhook) endpoint and send an empty `headers` object.
 
-Additionally, the HTTP reserved headers are blacklisted and cannot be used. You can see the full list of blacklisted header names below:
+```sh
+curl -X PUT \
+  https://api.fidel.uk/v1/hooks/3b4be60b-6596-4b40-ae3d-89b9fdaf132a \
+  -H 'content-type: application/json' \
+  -H 'fidel-key: sk_test_50ea90b6-2a3b-4a56-814d-1bc592ba4d63' \
+  -d '{
+    "programId": "06471dbe-a3c7-429e-8a18-16dc97e5cf35",
+    "event": "transaction.auth",
+    "url": "https://example.com",
+    "headers": {}
+  }'
+```
+
+A maximum of five custom headers per webhook can be defined, and they need to follow strict character validation patterns. The key name must be between 1 and 64 characters and only accepts the Roman alphabet, numbers, dashes and underscores. The value must be between 1 and 1000 characters. Additionally, the HTTP reserved headers are blocklisted and cannot be used for the key name. The full list of blocklisted key names:
 
 ```json
-[
-  "Accept",
-  "Accept-Charset",
-  "Accept-Encoding",
-  "Accept-Language",
-  "Accept-Datetime",
-  "Access-Control-Request-Method",
-  "Access-Control-Request-Headers",
-  "Cache-Control",
-  "Connection",
-  "Content-Length",
-  "Content-Type",
-  "Cookie",
-  "Date",
-  "Expect",
-  "Forwarded",
-  "From",
-  "Host",
-  "If-Match",
-  "If-Modified-Since",
-  "If-None-Match",
-  "If-Range",
-  "If-Unmodified-Since",
-  "Max-Forwards",
-  "Origin",
-  "Pragma",
-  "Proxy-Authorization",
-  "Range",
-  "Referer",
-  "Te",
-  "Transfer-Encoding",
-  "User-Agent",
-  "Upgrade",
-  "Via",
-  "Warning",
-  "Fidel-Account",
-  "Fidel-Key",
-  "Fidel-Live",
-  "Fidel-Request-Id",
-  "Fidel-User",
-  "X-Fidel-Signature",
-  "X-Fidel-Timestamp"
-]
+["Accept", "Accept-Charset", "Accept-Encoding", "Accept-Language", "Accept-Datetime", "Access-Control-Request-Method", "Access-Control-Request-Headers", "Cache-Control", "Connection", "Content-Length", "Content-Type", "Cookie", "Date", "Expect", "Forwarded", "From", "Host", "If-Match", "If-Modified-Since", "If-None-Match", "If-Range", "If-Unmodified-Since", "Max-Forwards", "Origin", "Pragma", "Proxy-Authorization", "Range", "Referer", "Te", "Transfer-Encoding", "User-Agent", "Upgrade", "Via", "Warning", "Fidel-Account", "Fidel-Key", "Fidel-Live", "Fidel-Request-Id", "Fidel-User", "X-Fidel-Signature", "X-Fidel-Timestamp"]
```

Comment:
I preferred this with newlines, this is not so readable inline

Reply:
My concern here is space. It takes too much space with new lines.